src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/FSharp.BrowseObject.xaml

```diff
@@ -0,0 +1,99 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright, Microsoft Corporation, All rights reserved.-->
```

Comment:
Actually, why can we just use the C# one? Don't we do that for VB? And just rename the thing to "Compile"